### PR TITLE
slim down repository

### PR DESCRIPTION
**Pull request by axe312ger**
* reference original community version in readme and make clear this is a slim copy of it
* remove any code reference to the original version
* remove dependencies irrelevant to onboarding
* add `WHATS-NEXT.md` for the last step in cli guide

Related PR in cli: https://github.com/contentful/contentful-cli/pull/57

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# gatsby-contentful-starter
+# Contentful Gatsby Starter Blog
 
-Gatsby [Contentful](https://www.contentful.com) starter for creating a blog
+Create a [Gatsby](http://gatsbyjs.com/) blog powered by [Contentful](https://www.contentful.com). This is a simplified version of the [Gatsby Contentful Starter](https://github.com/contentful-userland/gatsby-contentful-starter) maintained by our Community.
```
Comment (loudmouth) [inline README.md]: the wording _could_ possibly cause mild confusion here. Please rewrite to "which is maintained by our Community." at the end.

File: README.md [hunk 2]
```diff
@@ -11,70 +11,33 @@ With Contentful and Gatsby you can connect your favorite static site generator w
...
-* Responsive/adaptive images via [gatsby-image](https://www.gatsbyjs.org/packages/gatsby-image/)
-
-## Contribution
-
-This project is part of [contentful-userland](https://github.com/contentful-userland) which means that we’re always open to contributions and pull requests. You can learn more about how contentful userland is organized by visiting [our about repository](https://github.com/contentful-userland/about).
-
-## Requirements
-
-To use this project you have to have a Contentful account. If you don't have one yet you can register at [www.contentful.com/sign-up](https://www.contentful.com/sign-up/).
+* Using our [Delivery API](https://www.contentful.com/developers/docs/references/content-delivery-api/) via our [synchronization feature](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/synchronization).
```
Comment (loudmouth) [inline README.md]: "Use the synchronization feature of our Delivery API"

File: README.md [hunk 3]
```diff
@@ -11,70 +11,33 @@ With Contentful and Gatsby you can connect your favorite static site generator w
...
+See our [official Contentful get started guide](https://www.contentful.com/developers/docs/tutorials/general/get-started/).
 
 ## Crucial Commands
 
-This project comes with a few handy commands for linting and code fixing. The most important ones are the ones to develop and ship code. You can find the most important commands below.
-
 ### `npm run dev`
 
-Run in the project locally.
+Run in the project locally with live reload in development mode.
```
Comment (loudmouth) [inline README.md]: "Run the project..."

File: README.md [hunk 4]
```diff
@@ -11,70 +11,33 @@ With Contentful and Gatsby you can connect your favorite static site generator w
...
 
 ### `npm run build`
 
 Run a production build into `./public`. The result is ready to be put on any static hosting you prefer.
 
-### `npm run deploy`
+### `npm run serve`
 
-Run a production build into `./public` and publish the site to GitHub pages.
+Spin up a production ready server with your blog. Don't forget to build your page beforehand.
```
Comment (loudmouth) [inline README.md]: "production-ready"

File: README.md [hunk 5]
```diff
@@ -11,70 +11,33 @@ With Contentful and Gatsby you can connect your favorite static site generator w
...
 
-### `npm run cleanup-repository`
+## Deployment
 
-Removes all dependencies, scripts and data from the installation script.
+See the [official Contentful get started guide](https://www.contentful.com/developers/docs/tutorials/general/get-started/).
```
Comment (loudmouth) [inline README.md]: "getting started guide"

File: README.md [hunk 6]
```diff
@@ -11,70 +11,33 @@ With Contentful and Gatsby you can connect your favorite static site generator w
...
-## Roadmap
-
-- [x] [make the starter completely responsive](https://github.com/contentful-userland/gatsby-contentful-starter/issues/2)
-- [ ] [include tags](https://github.com/contentful-userland/gatsby-contentful-starter/issues/3)
-- [x] [support traced placeholders](https://github.com/contentful-userland/gatsby-contentful-starter/issues/4)
-- [ ] [add i18n](https://github.com/contentful-userland/gatsby-contentful-starter/issues/6)
+## Contribution
 
-## Other resources
+Feel free to open pull requests to fix bugs. If you wan't to add features, please have a look at the [original version](https://github.com/contentful-userland/gatsby-contentful-starter). It is always open to contributions and pull requests.
```
Comment (loudmouth) [inline README.md]: `wan't` -> `want`

File: README.md [hunk 7]
```diff
@@ -11,70 +11,33 @@ With Contentful and Gatsby you can connect your favorite static site generator w
...
 
-- Tutorial video series ["Building a blazing fast website with GatsbyJS and Contentful"](https://www.youtube.com/watch?v=Ek4o40w1tH4&list=PL8KiuH6vpACV-F7jXribe4YveGBhBeG9A) by @Khaledgarbaya
+You can learn more about how contentful userland is organized by visiting [our about repository](https://github.com/contentful-userland/about).
```
Comment (loudmouth) [inline README.md]: "contentful-userland" or "Contentful userland"

File: WHATS-NEXT.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+# Deploy the blog
+A blog on your local machine is nice to have, but a blog on the internet is even better. [Return to the tutorial doc, select a hosting service (Netlify or Heroku) and follow the CLI to deploy](https://www.contentful.com/developers/docs/tutorials/general/get-started/).
```
Comment (loudmouth) [inline WHATS-NEXT.md]: looks good. later, let's use the markdown anchor 👌 after we've updated the doc.

Comment (axe312ger) [reply WHATS-NEXT.md]: yeah exactly :)

Comment (axe312ger) [reply WHATS-NEXT.md]: I also want to see how the link will be formatted

**Pull request merged**